configuration/configuration.go

```diff
@@ -193,55 +192,52 @@ func (ep *EntryPoints) String() string {
 // Set's argument is a string to be parsed to set the flag.
 // It's a comma-separated list, so we split it.
 func (ep *EntryPoints) Set(value string) error {
-	result, err := parseEntryPointsConfiguration(value)
-	if err != nil {
-		return err
-	}
+	result := parseEntryPointsConfiguration(value)
 
 	var configTLS *TLS
-	if len(result["TLS"]) > 0 {
+	if len(result["tls"]) > 0 {
```

Comment:
WDYT about using constants for all fields?

Reply:
use constant for 2 strings seems overkill

Reply:
I'm talking about all the strings used in entrypoint definition (`tls`, `redirect_entrypoint`...) because they are used in many places, in many files.
IMHO, it's easier to modify one place.